diff --git a/gst-sample-apps/CMakeLists.txt b/gst-sample-apps/CMakeLists.txt
index 0ed9692c..e525f75c 100644
--- a/gst-sample-apps/CMakeLists.txt
+++ b/gst-sample-apps/CMakeLists.txt
@@ -34,12 +34,12 @@ if (ENABLE_CAMERA AND ENABLE_DISPLAY AND ENABLE_VIDEO_ENCODE)
   add_subdirectory(gst-multi-stream-example)
   add_subdirectory(gst-add-streams-as-bundle-example)
   add_subdirectory(gst-add-remove-streams-runtime)
-  add_subdirectory(gst-camera-burst-capture-example)
+  #add_subdirectory(gst-camera-burst-capture-example)
 endif () # ENABLE_CAMERA AND ENABLE_DISPLAY AND ENABLE_VIDEO_ENCODE

 if (ENABLE_CAMERA)
   add_subdirectory(gst-appsink-example)
-  add_subdirectory(gst-camera-metadata-example)
+  #add_subdirectory(gst-camera-metadata-example)
   add_subdirectory(gst-camera-switch-example)
 endif () # ENABLE_CAMERA

